pl.art.lach.slawek.V2BlankBrowser.json

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "pl.art.lach.slawek.V2BlankBrowser",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "41",
+  "sdk": "org.gnome.Sdk",
+  "command": "V2BlankBrowser",
+  "clear": [ "*.o"],
```

Comment:
there is no such property.
```suggestion
```

Reply:
From the build logs:

`(flatpak-builder:3975308): flatpak-builder-WARNING **: 00:08:51.099: Unknown property clear for type BuilderManifest`